primitives/api/proc-macro/src/mock_impl_runtime_apis.rs

```diff
@@ -280,7 +280,7 @@ impl<'a> Fold for FoldRuntimeApiImpl<'a> {
 			// Rewrite the input parameters.
 			input.sig.inputs = parse_quote! {
 				&self,
-				___at___param___: #block_id_type,
+				#at_param_name: #block_id_type,
 				_: #crate_::ExecutionContext,
 				params: Option<( #( #param_types ),* )>,
```

Comment:
(I'm ok just, this can mae `#at_param_name` collide with `params` thus wouldn't compile if user do `fn foo(params: BlockId<...>)` )